data/tests/tests_ship_sale.txt

```diff
@@ -0,0 +1,49 @@
+test "Sell 1 ship"
+	status active
+	description "Test selling a single ship."
+	sequence
+		# Create/inject the savegame and load it.
+		inject "Fighters and Carriers and Escorts"
+		call "Load First Savegame"
+		call "Sell first ship"
+		call "Depart"
+		assert
+			and
+				"ships: Fighter" == 1
+				"ships: Heavy Warship" == 1
+				"ships: Medium Warship" == 1
```

Comment:
I'm used to test always test preconditions as well and it's a good practice too I believe. So that there are 2 fighters, 1 HW and 1 MW after loading the first save game. Same for the other tests below.

Reply:
Yes, that is a good practice, but the "update_auto_conditions" functions currently don't run directly after loading, and they also don't run after buying or selling ships. So the numbers can only be checked after departing (the numbers are not correct before departing).